scripts/tpu_resource.sky.yaml

```diff
@@ -16,3 +16,6 @@ setup: |
   uv pip install -e "python[all]"
   bash scripts/killall_sglang.sh
   uv tool install evalscope
+  python -m ensurepip --upgrade
+  python -m pip install --upgrade pip
+  pip install git+https://github.com/google/qwix
```

Comment:
`qwix` is already included in pyproject; does that mean we don’t need to add it in skypilot init?